documentation/src/main/asciidoc/topics/ref_discovery_google.adoc

```diff
@@ -1,16 +1,22 @@
-= GOOGLE2_PING
-On Google Compute Engine (GCE), use a generic discovery protocol or
-`GOOGLE2_PING`, which uses Google Cloud Storage (GCS) to store information
-about the cluster members.
+[id='discovery_google-{context}']
+= GOOGLE_PING2
+On Google Compute Engine (GCE), use a generic discovery protocol or GOOGLE_PING2, which uses Google Cloud Storage (GCS) to store information about the cluster members.
 
-.GOOGLE2_PING configuration example
+.GOOGLE_PING2 configuration example
 [source,xml,options="nowrap",subs=attributes+]
 ----
 include::config_examples/googleping.xml[]
 ----
 
-.GOOGLE2_PING dependencies for embedded {brandname}
+ifdef::embedded_mode[]
+{brandname} requires cloud provider dependencies to use GOOGLE_PING2:
```

Comment:
I'm overthinking again :fearful: 

this section is fine for embedded. But for the server, it requires custom jars to be placed in `server/lib`: the `jgroups-google` jar and manually find its dependencies. (same for azure & native_s3)

this isn't a trivial task and the server docs make it look too easy. I'm not sure how to proceed here:
* Remove those discovery protocols from server docs?
* something else?

Reply:
ps. actually, we don't ship the native_s3 for 11.x/master. We ship in the old server 9.4.x.

Reply:
@pruivo Let's remove the discovery protocols from server docs. I'll remove NATIVE_S3 altogether.

Reply:
As discussed, let's drop the 3 cloud protocols. Easier to maintain for us and we link to JGroups docs.